Improve swap tutorial (#211)
cowprotocol/docs: docs/cow-protocol/tutorials/cow-swap/swap.mdx

<Image img={require('/img/cowswap/swap_need_approval.png')} alt="GPv2VaultRelayer approval required" />

> When you do a swap on CoW Swap for the first time, you must give approval for the [Vault relayer contract](/cow-protocol/reference/contracts/core/vault-relayer) contract to spend your token (in this case, `WETH`). 

Reviewer: ```suggestion
When you do a swap on CoW Swap for the first time, you must give approval for the [_vault relayer_](/cow-protocol/reference/contracts/core/vault-relayer) contract to spend your token (in this case, `WETH`). 
```

> This is because this contract needs to pull the `WETH` from our wallet and send it to the [Settlement contract](/cow-protocol/reference/contracts/core/settlement) that will co-ordinate the swap on your behalf.

Reviewer: ```suggestion
This is because this contract needs to pull the `WETH` from our wallet and send it to the [_settlement_](/cow-protocol/reference/contracts/core/settlement) contract that will co-ordinate the swap on your behalf.
```

Both contracts are audited, battle-tested and non-upgradeable. 

> Moreover, the settlement contract is the only address that can request the relayer to pull your funds and **only after verifying** both that:

Reviewer: ```suggestion
Moreover, the settlement contract is the only address that can request the vault relayer to pull your funds and **only after verifying** both that:
```

1. Your order was signed by you
2. and that you will receive at least your limit execution price or better

## Approve sell token

:::tip

> Some tokens, such as `USDC`, `DAI`, and `COW` support permit-style approvals. This means that you can approve the [Vault relayer contract](/cow-protocol/reference/contracts/core/vault-relayer) to spend your tokens without having any initial gas for an on-chain transaction. This is a great way to save on gas costs!

Reviewer: ```suggestion
Some tokens, such as `USDC`, `DAI`, and `COW` support permit-style approvals. This means that you can approve the [vault relayer](/cow-protocol/reference/contracts/core/vault-relayer) contract to spend your tokens without having any initial gas for an on-chain transaction. This is a great way to save on gas costs!
```

You don't have to worry about this, as CoW Swap will automatically detect if the token supports permit-style approvals and will guide you through the process.

:::

<Image img={require('/img/cowswap/swap_approval_default.png')} alt="GPv2VaultRelayer default approval" />

> By default, CoW Swap prompts you to give [Vault relayer contract](/cow-protocol/reference/contracts/core/vault-relayer) an "unlimited" allowance for the sell token.

Reviewer: ```suggestion
By default, CoW Swap prompts you to give the [vault relayer](/cow-protocol/reference/contracts/core/vault-relayer) contract an "unlimited" allowance for the sell token.
```

<Image img={require('/img/cowswap/swap_signing.png')} alt="Signing Swap" />

> Once the order has been signed, it will be submitted to CoW Protocol and ready to be executed! 🎉🎉🎉🎉

Reviewer: ```suggestion
Once the order has been signed, it will be submitted to CoW Protocol where it is now ready to be executed! 🎉
```

<Image img={require('/img/cowswap/swap_confirmed.png')} alt="Swap confirmed" />

> ## Follow up on your order

Reviewer: ```suggestion
## Track your order
```
